# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/clucene/hotfix-ctime.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- clucene-core-2.3.3.4/src/core/CLucene/document/DateTools.h.vanilla	2025-01-23 17:00:45.670002519 +0100
+++ clucene-core-2.3.3.4/src/core/CLucene/document/DateTools.h	2025-01-23 17:00:54.016669183 +0100
@@ -22,6 +22,8 @@
 # include <sys/timeb.h>
 #endif
 
+#include <ctime>
+
 CL_NS_DEF(document)
 
 class CLUCENE_EXPORT DateTools {
